PhysicsTools/PatAlgos/python/slimming/packedPFCandidates_cfi.py

```diff
@@ -6,5 +6,8 @@
     originalVertices = cms.InputTag("offlinePrimaryVertices"),
     originalTracks = cms.InputTag("generalTracks"),
     vertexAssociator = cms.InputTag("primaryVertexAssociation","original"),
+    PuppiWeight = cms.InputTag("puppi","PuppiWeights"),
+    PuppiSrc = cms.InputTag("puppi"),
+    PuppiSrcMap = cms.InputTag("puppi"),
```

Comment:
we do not need two input tags here, one is enough as the producer is expected to be the same (in fact, is the label "PuppiWeights" really needed ???)